build-package.sh

```diff
@@ -10,12 +25,24 @@ export TERMUX_SCRIPTDIR
 # Store pid of current process in a file for docker__run_docker_exec_trap
 source "$TERMUX_SCRIPTDIR/scripts/utils/docker/docker.sh"; docker__create_docker_exec_pid_file
 
+# Functions for working with packages
+source "$TERMUX_SCRIPTDIR/scripts/utils/package/package.sh"
 
 SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct 2>/dev/null || date "+%s")
 export SOURCE_DATE_EPOCH
 
-: "${TMPDIR:=/tmp}"
-export TMPDIR
+# Check if the package is in the compiled list
+termux_check_package_in_built_packages_list() {
+	[ -f "$TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH" ] && cat "$TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH" | grep " $1 " &>/dev/null
```

Comment:
Don't check existence like I mentioned before. The `build-package.sh` should abort if file doesn't exist like in case something accidentally deleted it while it was still needed.

Reply:
And should use `grep -q` instead of redirect errors (and output) to `/dev/null`.